packages/portal-contracts/src/Portal.sol

```diff
@@ -366,6 +378,11 @@ contract Portal is Owned {
         if (o.amountSats == 0) {
           delete orderbook[orderID];
         }
+	
+	bytes32 recKey = recipientKey(destScriptHash, amountSats);;
+	int256 storage existingRecipient = recipients[recKey]
+	assert(existingRecipient == 0, "Must send a unique amountSats to recipient compared to inflight escrows");
+        recipients[recKey] = btcVerifier.getLatestBlockHeight();
```

Comment:
Subtle issue.

Say Alice opens an escrow at block height 1000. She submits a Bitcoin transaction. A normal two-block reorg occurs, and her transaction ends up confirmed at block height 999.

We can fix that as follows:

```
recipients[recKey] = btcVerifier.mirror.getLatestBlockHeight() - minConfirmations + 1;
```

...with an explanation coment.

+ Update the comment on the `recipients` field to say it's the min Bitcoin block height at which that escrow can be paid.

It may be worth renaming `recipients` to `openEscrows`.